README.md

```diff
@@ -204,7 +235,6 @@ Connector class name, in this case: `io.aiven.kafka.connect.s3.AivenKafkaConnect
 ### S3 Object Names
 
 S3 connector stores series of files in the specified bucket. Each object is named using pattern `[<aws.s3.prefix>]<topic>-<partition>-<startoffset>[.gz]`. The `.gz` extension is used if gzip compression is used, see `file.compression.type` below.
```

Comment:
Pattern is missing the versions where key is used. The section "File name format" would be enough?